### DISCUSSION TITLE
怎么将正在使用的 iconfont 导出后导入到文档项目中

### DISCUSSION THREAD

**Discussion by ghost** [vuepress-theme-hope/vuepress-theme-hope#3165] [Q&A] [answered]
如题

**Comment by Mister-Hope** [comment, marked as answer]
https://theme-hope.vuejs.press/zh/guide/interface/icon.html#%E7%94%9F%E6%88%90%E8%87%AA%E5%B7%B1%E7%9A%84-iconfont-%E8%B5%84%E6%BA%90
